src/todoListItem.js

```diff
@@ -1,8 +1,18 @@
-import React from "react";
+import React ,{useState} from "react";
+import style from "./TodoListItem.module.css"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrashCan } from '@fortawesome/free-solid-svg-icons'
 
-function TodoListItem({todo,onRemoveTodo}){
+function TodoListItem({ todo, onRemoveTodo,checkBox}) {
+ 
     return (
-        <li>{todo.title} <button onClick={()=>onRemoveTodo(todo.id)}>Remove</button></li>
+        <>
+            <li className={style.ListItem} style={{ backgroundColor: todo.done ? 'rgb(44, 162, 44)' : 'white'}}>
```

Comment:
Great usage of state driving a style! On a side note, you could also this toggle a class instead of a style:

```jsx
<li className={`${style.ListItem} ${todo.done ? style.TodoCompleted : style.TodoActive}`}> 
  ... 
</li>
```

```css
.TodoActive {
  background: rgb(255, 255, 255);
}

.TodoCompleted {
  background: rgb(44, 162, 44);
}
```